modules/nw-route-specific-annotations.adoc

```diff
@@ -53,3 +54,34 @@ metadata:
 Setting a server-side timeout value for passthrough routes too low can cause
 WebSocket connections to timeout frequently on that route.
 ====
+
+
+.A route rewrite-target annotation
+[source,yaml]
+----
+apiVersion: v1
+kind: Route
+metadata:
+  annotations:
+    haproxy.router.openshift.io/rewrite-target: / <1>
+...
+----
+<1> Sets / as rewrite path of the request on the backend
+
+.rewrite-target examples:
+[cols="4*", options="header"]
+|===
+|Route.spec.path|Request path|Rewrite target| Output
+|/foo|/foo|/|/
+|/foo|/foo/|/|/
+|/foo|/foo/bar|/|/bar
+|/foo|/foo/bar/|/|/bar/
+|/foo|/foo|/bar|/bar
+|/foo|/foo/|/bar|/bar/
+|/foo|/foo/bar|/baz|/baz/bar
+|/foo|/foo/bar/|/baz|/baz/bar
```

Comment:
Are we saying we'll be removing trailing `/` ? If yes, isn't it inconsistent with line 78?

Reply:
I made an error when recopying result, fixed